lib/libutee/tee_api.c

```diff
@@ -14,6 +14,11 @@
 
 static const void *tee_api_instance_data;
 
+static void *tee_null_sized_va(void)
+{
+	return (void *)1;
+}
+
```

Comment:
Why not simply `#define NULL_SIZED_VA (void *)1`?

Reply:
tried to keep it modular for readability purpose

Reply:
I find the function slightly *less* readable if anything...

Reply:
No problem - shall change it

Reply:
fine